### ISSUE TITLE
Empty cells not rending styles

### ISSUE THREAD

**Issue by yesman85** [protobi/js-xlsx#23]
I have a workbook that has some cells bordered but there is not content in it.

When I save the document these cells are completely cleared out, is there a way to keep those cells that have no content, just styling?

**Comment by pietersv** [comment]
Concur this is an issue. It goes a bit deeper as the root project before had little need to write empty cells.  Am hesitant to modify until I get the tests running again, as that will surely affect some case out there.  Possible work around is setting the contents to a single space, so the row is not blank but appears that way, `{v: ' '}`

**Comment by yesman85** [comment]
Yeah I did that actually now, I put spaces in the cells I need rendered, it works, but you need to know what you're doing :)
